core/src/main/java/io/cucumber/core/api/event/EmbedEvent.java

```diff
@@ -9,10 +11,17 @@ public EmbedEvent(Long timeStamp, TestCase testCase, byte[] data, String mimeTyp
         this(timeStamp, 0, testCase, data, mimeType);
     }
 
+    @Deprecated
     public EmbedEvent(Long timeStamp, long timeStampMillis, TestCase testCase, byte[] data, String mimeType) {
```

Comment:
No need to deprecate this. It can be removed all together.

Reply:
Same for all the other event constructors.

Reply:
You mean remove all deprecations from `TestCaseStarted`, `TestCaseFinished`, `TimeStampedEvent`, etc  also?

Since they all extend `Event`

Reply:
You can remove all those constructors. This is v5 so no need to deprecate them first.